### PR TITLE
fix #61065 politico.com

### LINKED ISSUE #61065: politico.com
### Issue URL (Anti Adblock Script)
 [https://www.politico.com/cartoons/2020/08/07/the-nations-cartoonists-on-the-week-in-politics-000161?slide=0](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.politico.com%2Fcartoons%2F2020%2F08%2F07%2Fthe-nations-cartoonists-on-the-week-in-politics-000161%3Fslide%3D0)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v1.7.4
Browser: | Safari
Stealth mode options: | 
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Security:</b><br/> Malware Domains Blocklist, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard DNS

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#61065

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1484,6 +1484,11 @@ oneplus.com,tickets.ua,woman.ru,elle.ru,marieclaire.ru,europaplus.ru,krossovki.c
 beautyinsider.ru###articleSubscribeBlock
 officelife.media##.subscribe-plate
 pinsknews.by#?#.sidebar_list > li.widget:has(> div.textwidget > form[action] strong:contains(Подпишитесь))
+politico.com#?#.below-article-section div.below-story-text:has(> aside > section > div.sign-up__column > div.sign-up__summary > a[href="https://www.politico.com/newsletters"])
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:1487]: Not `.below-article-section div.below-story-text > aside:has(`? Current rule blocks too much.
https://www.politico.com/news/2020/08/07/rashida-tlaib-campaign-finance-violation-392579

File: AnnoyancesFilter/sections/annoyances.txt [hunk 2]
```diff
@@ -1484,6 +1484,11 @@ oneplus.com,tickets.ua,woman.ru,elle.ru,marieclaire.ru,europaplus.ru,krossovki.c
...
+politico.com#?#.container div.story-text > aside:has(> section > div.sign-up__column > div.js-tealium-newsletter)
+politico.com#?#.container .container__slot > section:has(> div.sign-up__column > div.js-tealium-newsletter)
+politico.com##aside[class*="morning"][class*="esu"]
+politico.com##.morning-energy
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:1491]: where these rules must be applied?

File: EnglishFilter/sections/whitelist.txt [hunk 3]
```diff
@@ -230,6 +230,10 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 @@||yastatic.net/pcode/adfox/loader.js$domain=onedio.ru
 @@||yastatic.net/pcode/adfox/header-bidding.js$domain=onedio.ru
 !#endif
+! https://github.com/AdguardTeam/AdguardFilters/issues/61065
+@@||securepubads.g.doubleclick.net/tag/js/gpt.js$domain=politico.com
+@@||securepubads.g.doubleclick.net/gpt/pubads_impl_*.js$domain=politico.com
```
Comment (Alex-302) [inline EnglishFilter/sections/whitelist.txt:235]: is it a fix for slider?
https://www.politico.com/cartoons/2020/08/07/the-nations-cartoonists-on-the-week-in-politics-000161?slide=7

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/whitelist.txt (hunk 3 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/whitelist.txt:235]: Yes

File: AnnoyancesFilter/sections/annoyances.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/annoyances.txt:1487]: You are right

File: AnnoyancesFilter/sections/annoyances.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/annoyances.txt:1491]: for example
Screenshot 2020-08-09 at 15 27 48

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/annoyances.txt:1491]: Screenshot 2020-08-09 at 15 29 52

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/annoyances.txt:1491]: Screenshot 2020-08-09 at 15 35 06

File: EnglishFilter/sections/whitelist.txt (hunk 3 above)
Comment (uBlock-user) [reply EnglishFilter/sections/whitelist.txt:235]: alternative fix without exceptions -- `politico.com##+js(set, generateGalleryAd, noopFunc)`

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/whitelist.txt:235]: > alternative fix without exceptions -- `politico.com##+js(set, generateGalleryAd, noopFunc)`

It works, thank you.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**